catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/AimidbAttribute.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.transformer.nitf.common;
+
+import java.io.Serializable;
+import java.util.function.Function;
+
+import org.codice.imaging.nitf.core.tre.Tre;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.types.LocationAttributes;
+import ddf.catalog.data.types.Location;
+
+public enum AimidbAttribute implements NitfAttribute<Tre> {
+    COUNTRY(Location.COUNTRY_CODE,
```

Comment:
According to the spec, country is a two letter code defining the country for the reference point of the image. I do not believe it can have multiple country codes.